tests/nosetests/pyanaconda_tests/simple_ui_test.py

```diff
@@ -110,6 +129,16 @@ def tui_test(self, proxy_getter):
             ]
         })
 
+        # Force us to always decide order of standalone spokes based on priority not by name.
+        # This will ordering errors easier to spot.
+        spokes = self.ordered_action_classes
+
+        for hub in self.hubs:
+            pre_spokes = filter(lambda obj, h=hub: getattr(obj, "preForHub", None) == h, spokes)
```

Comment:
I don't understand why we define `h=hub` in these lambda functions.

Reply:
I've just copied this. I guess it's some old convention?

Reply:
Fixed.